includes/Data/Patterns.php

```diff
@@ -6,6 +6,32 @@
  */
 final class Patterns {
 
+	/**
+	 * List of styles to check for header menu slugs and replace with user selected header menu slug.
+	 *
+	 * @var array
+	 */
+	private static $styles_to_check_for_header_menu = array(
+		'theme-styles',
+		'homepage-styles',
+	);
+
+	/**
+	 * List of dummy menu page titles.
+	 *
+	 * @return array
+	 */
+	protected static function default_menu_items() {
```

Comment:
Can we move `default_menu_items` and `dummy_pages` to a single file/class?

Reply:
Done ... referencing the default_menu_items everywhere else